app/src/main/java/org/piwigo/ui/about/AboutViewModel.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Piwigo for Android
+ * Copyright (C) 2016-2017 Piwigo Team http://piwigo.org
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.piwigo.ui.about;
+
+import android.arch.lifecycle.ViewModel;
+import android.content.res.Resources;
+import android.text.Html;
+import android.text.Spanned;
+import android.text.method.LinkMovementMethod;
+import android.widget.TextView;
+
+import org.piwigo.BuildConfig;
+import org.piwigo.R;
+
+public class AboutViewModel extends ViewModel {
+    private AboutActivity aboutActivity;
+    private final Resources resources;
+
+    public AboutViewModel(Resources resources) {
+        this.resources = resources;
+    }
+
+    public void setView(AboutActivity act) {
```

Comment:
`ViewModel` lifecycle outlives `Activity`, this will leak.

Reply:
Don't remember where I got this from, but the reason why I put a TODO there was because I had a bad feeling about doing that - a reference from the ViewModel into the activity seems not a clean approach. And in this case looks complete not needed...

Nevertheless I'd like to understand where you see the memory leak. COuld you elaborate?

Reply:
If you google for something like "android leaked context" you should find plenty of info.
Essentially the reference in the view model could prevent GC from running on the activity.

Reply:
I was not really aware of the leaked context issue and I think I understood the point. Still in the code I do not see the real leak, because the ViewModel should not live longer than the activity - which is the only object with a reference to it, and in case of re-creation e. g. by screen orientation change, the reference from the view to the activity should be updated, shouldn't it? In case I am wrong, please open my eyes!

Beside that I agree, that having a reference in the ViewModel to the view doesn't seem clean and for the AboutView there is even no real use, so I get rid of it.